components/dynapictures/actions/create-image/create-image.mjs

```diff
@@ -0,0 +1,52 @@
+import app from "../../dynapictures.app.mjs";
+import utils from "../../common/utils.mjs";
+
+export default {
+  key: "dynapictures-create-image",
+  name: "Create Image",
+  description: "Generates a new image by using a given image as a template. [See the documentation](https://dynapictures.com/docs/#image-generation)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    app,
+    templateId: {
+      propDefinition: [
+        app,
+        "templateId",
+      ],
+    },
+    params: {
+      type: "string[]",
+      label: "Image Parameters",
+      description: "List of custom parameters, specifying new values for the image layers. It's possible to replace text and images, adjust styling, etc. For each element in the list, specify a JSON object with the following properties as an example: ```{ \"name\": \"title\", \"text\": \"Lorem ipsum\" }```. See details [here](https://dynapictures.com/docs/#image-generation).",
+    },
```

Comment:
I'll leave this up to you, but I wonder if it would be a better user experience to have the image params (name, text, color, etc.) listed individually as optional props?